templates/Default/engine/Default/game_join.php

```diff
@@ -6,7 +6,7 @@
 	<h2>Round Description</h2>
 	<p><?php echo bbifyMessage($Game->getDescription()); ?></p><?php
 }?>
-
+<script src='https://cdn.plot.ly/plotly-latest.min.js'></script>
```

Comment:
Two minor issues:
1. We are trying to adopt a consistent style of using double quotes in HTML (instead of single quotes).
2. Let's fix to a specific version of plotly so that we can control when we update to a new library (otherwise the graph could break when a new version is released).